diff --git a/toolkit/mozapps/update/updater/moz.build b/toolkit/mozapps/update/updater/moz.build
--- a/toolkit/mozapps/update/updater/moz.build
+++ b/toolkit/mozapps/update/updater/moz.build
@@ -80,6 +80,9 @@ elif CONFIG["MOZ_UPDATE_CHANNEL"] in (
 ):
     primary_cert.inputs += ["nightly_aurora_level3_primary.der"]
     secondary_cert.inputs += ["nightly_aurora_level3_secondary.der"]
+elif CONFIG["MOZ_UPDATE_CHANNEL"] == "nightsedge":
+    primary_cert.inputs += ["nightsedge_primary.der"]
+    secondary_cert.inputs += ["nightsedge_secondary.der"]
 else:
     primary_cert.inputs += ["dep1.der"]
     secondary_cert.inputs += ["dep2.der"]
